tests/sentry/issues/test_escalating.py

```diff
@@ -246,11 +235,16 @@ def test_daily_count_query(self) -> None:
             )
             group = event.group
 
-        # The group had 2 events yesterday
-        one_day_ago_mins = 24 * 60
+        # The group had 2 events yesterday, 10 mins before midnight
+        # Tests that events are aggregated in the daily count query by date, not by 24 hr periods
+        now = datetime.now()
+        mins_since_ten_mins_before_today = (
```

Comment:
Create a variable for last times midnight and use that as the base variable from which to build other variables. You will need it for my comment about this being an intermittent test if the CI was to execute around UTC midnight.

Reply:
thank you for this suggestion!